tests/api/api/test_projects.py

```diff
@@ -89,12 +90,23 @@ def test_get_non_existing_project(
 
 
 def test_delete_project_with_resources(
-    db: Session, client: TestClient, project_member_mode: str
+    db: Session,
+    client: TestClient,
+    project_member_mode: str,
+    k8s_secrets_mock: tests.api.conftest.K8sSecretsMock,
 ):
+    # need to set this to False, otherwise impl will try to delete k8s resources, and will need many more
+    # mocks to overcome this.
+    k8s_secrets_mock.set_is_running_in_k8s_cluster(False)
```

Comment:
I'm not sure I understand why it will require may more mocks, please explain (can be offline if easier)

Reply:
Let's take it offline.